### ISSUE TITLE
Creating a domain with admin password results in values do not match and failure PAYARA-3676

### ISSUE THREAD

**Issue by OndroMih** [payara/Payara#3833]
# Description #
----------

This is for Payara Server 5.191. Doesn't happen in 5.184. Linux OS (Ubuntu Bionic LTS)

When creating a new domain with `asadmin create-domain` and typing admin password, the command prints "Enter the admin password" again after the first key press. When asked to enter the password again, reports that the passwords don't match even if they are the same.

## Expected Outcome

* The `create-domain` command executes successfully
* A new domain `newdomain` is created
* the new domain is secured by admin user with password `password`
* running `bin/asadmin list-applications` asks for password, when `password` entered, executes successfully

## Current Outcome

* the command prompts for admin password
* after first key press, the command prints the prompt message again
* after entering full password, the command asks for entering the password again
* the same password is entered again
* the command complains that "CLI173: Values entered for Enter the admin password do not match."
* the command results in a failure

This is a log from the command executed in a terminal:

```
> bin/asadmin create-domain newdomain     
Enter admin user name [Enter to accept default "admin" / no password]>Enter the admin password [Enter to accept default of no password]> 
Enter the admin password again> 
CLI173: Values entered for Enter the admin password do not match.
Usage: asadmin [asadmin-utility-options] create-domain
	[--adminport <adminport(default:4848)>] [--portbase <portbase>]
	[--template <template>] [--domaindir <domaindir>]
	[--instanceport <instanceport(default:8080)>]
	[--savemasterpassword[=<savemasterpassword(default:false)>]]
	[--usemasterpassword[=<usemasterpassword(default:false)>]]
	[--domainproperties <domainproperties>]
	[--keytooloptions <keytooloptions>]
	[--savelogin[=<savelogin(default:false)>]]
	[--nopassword[=<nopassword(default:false)>]]
	[--checkports[=<checkports(default:true)>]]
	[-?|--help[=<help(default:false)>]] domain_name
Command create-domain failed.
```

## Steps to reproduce (Only for bug reports) 

* run `bin/asadmin create-domain newdomain`
* when asked for admin password, type password `password`
* when asked to re-enter the password again, type password `password` again

### A workaround

A workaround is to press any key before entering the password again. Pressing any key will trigger printing the prompt message again. What is typed from this point is taken as a password.

In other words, if the password is "password", then type "xpassword" when first asked for a password (the command will print the prompt message after pressing x). Then enter password "password" when asked to enter the password again.

## Context (Optional)

This happens when creating a new domain interactively and entering admin password when prompted. It's probably caused by an upgrade to JLink.

## Environment ##

- **Payara Version**: 5.191
- **Edition**: Full
- **JDK Version**: OpenJDK 8
- **Operating System**: Linux (Ubuntu Bionic)
